main.py

```diff
@@ -1,8 +1,18 @@
 import logging
 
+import mlflow
+import pandas as pd
 import yaml
+from databricks.connect import DatabricksSession
+from mlflow.models import infer_signature
 
 from diabetes.data_processor import DataProcessor
+from diabetes.diabetes_model import DiabetesModel
+
+mlflow.set_tracking_uri("databricks")
+mlflow.set_registry_uri("databricks-uc")
+
+spark = DatabricksSession.builder.getOrCreate()
```

Comment:
SparkSession is preferable over DatabricksSession - as DatabricksSession relies on specific version of databricks connect